nss-day-cohort-48/rare-client-orchesta-outlaws
# view my posts feature

#### Changes Made

1. Added files `PostDetail.js` and `PostForm.js` to `post` directory.
2. Modified file `ApplicationViews` to include routes to PostDetail and PostForm.
3. Modified `MyPost` icons to have click functionality.
4. Modified `MyPost` post title to link to `PostDetail`
5. Added styling to link objects

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout branchname
   ```
2. Ensure the backend and client are both running.
4. Test app functionality.
   
   >In client directory, run npm install react-bootstrap bootstrap@4.6.0
   >If you haven't already, also run npm install react-icons --save
   > Navigate to My Posts
   > click on the Post Title.  User should be redirected to posts/detail/{post.id}.  A heading should display that reads "Post Id {post.id}"
   > click on the Add Post icon.  User should be redirected to posts/create.  A heading should display that reads "New Post."
   > click on the Edit (gear) icon.  User should be redirected to posts/edit/{post.id}.  A heading should display that reads "Edit Post"
   > click on the Delete (trash) icon.  A modal box should appear asking if the user is sure they would like to delete the post as well as two buttons with the options 'YES' and 'CANCEL'.  Selecting either button will remove the modal box.  (Delete Post functionality has not been created yet)  
   
5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
